README.rst

```diff
@@ -43,10 +43,32 @@ If you are a core developer willing to upload datasets to the cloud you need to
 Usage
 -----
 
-Example 1: How do I download the datasets?
+We have `plenty of them <https://github.com/okfn-brasil/serenata-de-amor/blob/master/research/CONTRIBUTING.md#datasets-data>`_ ready for you to download from our servers. And this toolbox helps you get them. Here some examples:
+
+Example 1: Using the command line wrapper
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: bash
+
+    # without any arguments will download our pre-processed datasets and store into data/ folder
+    $ serenata-toolbox
+
+    # will download these specific datasets and store into /tmp/serenata-data folder
+    $ serenata-toolbox /tmp/serenata-data --module federal-senate reimbursements
+
+    # you can specify a dataset and a year
+    $ serenata-toolbox --module reimbursements --year 2009
+
+    # or specify all options simultaneously
+    $ serenata-toolbox /tmp/serenata-data --module federal-senate --year 2017
```

Comment:
All these instructions contrast with lines 29 and 35 from `serenata_toolbox/serenata-toolbox`. There's no `federal-senate` or `reimbursements` to be entered in the CLI anymore.

Reply:
Whoops. Fixing.